SyntaxFunctionsStatementsExercise/02-greatest-common-divisor.js

```diff
@@ -0,0 +1,3 @@
+function gcd(num1, num2) {
+    return !num2 ? num1 : gcd(num2, num1 % num2);
```

Comment:
```suggestion
    return num2 === 0 ? num1 : gcd(num2, num1 % num2);
```
Way easier to understand what is going on.